samples/openapi3/client/petstore/python-experimental/petstore_api/api/farm_api.py

```diff
@@ -0,0 +1,373 @@
+# coding: utf-8
```

Comment:
Can you delete this file after you switch the api tag to put the farm endpoint into the fake api?